back/CinemaReservation.BusinessLayer/Models/ResultStatus.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CinemaReservation.BusinessLayer.Models
+{
+    public enum ResultStatus
+    {
+        UserExists,
```

Comment:
It is worth to place something general as first enum member because it is used as default value. Ok is better than user exists. Or None value is also good as first item